src/backend/distributed/citus--6.1-17--6.1-18.sql

```diff
@@ -0,0 +1,24 @@
+/* citus--6.1-17--6.1-18.sql */
+
+SET search_path = 'pg_catalog';
+
+ALTER TABLE pg_dist_node ADD COLUMN noderole "char" NOT NULL DEFAULT 'p';
```

Comment:
I wanted to, however that's not allowed, [apparently](https://github.com/postgres/postgres/blob/master/src/backend/catalog/indexing.c#L114).